Source/automap.cpp

```diff
@@ -45,10 +47,46 @@ static Sint32 AutoMapY;
 #define MAPFLAG_DIRT 0x40
 #define MAPFLAG_STAIRS 0x80
 
+/**
+ * @brief Renders a shape for entrance to next level at the specified screen coordinates.
+ */
+static void DrawEntranceNext(CelOutputBuffer out, Sint32 sx, Sint32 sy)

```

Comment:
The 4x `DrawLineTo` Sourcecode seams to be always identical.
Could we pass the color as Parameter to the method?
Rename it to DrawEntrace and use it everywhere (also at the original location).

Anyway, I like the changes. 🙂

Reply:
Sure, but i kind of wanted it to be a placeholder for defining custom shapes later if we manage to come to some agreement 😅